### ISSUE TITLE
Double // @flow comment added to generated file

### ISSUE THREAD

**Issue by FezVrasta** [dotansimha/graphql-code-generator#4911]
**Describe the bug**
 

Hi, when I enable `flow` and `flow-operations` plugins at the same time, two `// @flow` headers are added to the `graphql.js` file.

**To Reproduce**

I tried to edit the template but it doesn't seem to work with the Flow plugins.

https://codesandbox.io/s/wizardly-paper-wsqgl?file=/document.graphql

1. My GraphQL schema:

```graphql
Not relevant for the issue
```

2. My GraphQL operations:

```graphql
Not relevant for the issue
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: 'allSchema.graphql'
generates:
  ./src/generated/graphql.js:
    documents: 'queries/*.js'
    plugins:
      - flow
      - flow-operations
```

**Expected behavior**

**Environment:**

- OS: macOS Catalina
- `@graphql-codegen/flow`: same as codesandbox
- `@graphql-codegen/flow-operations`: same as codesandbox
- NodeJS: 14.5.0

**Additional context**

**Comment by dotansimha** [comment]
I think this was introduced in: https://github.com/dotansimha/graphql-code-generator/pull/4609  
Does it's breaking anything @FezVrasta ? or it's just semantics?

**Comment by FezVrasta** [comment]
It errors in my create-react-app project

**Comment by dotansimha** [comment]
@FezVrasta  build error or lint error? As far as I can tell, the Flow compiler works perfectly fine with that...

**Comment by FezVrasta** [comment]
![image](https://user-images.githubusercontent.com/5382443/96447775-bb899780-1212-11eb-8564-bdbaaccde23e.png)

**Comment by sam-rad** [comment]
Having the the same issue. This is where [flow throws](https://github.com/facebook/flow/blob/master/src/typing/errors/error_message.ml#L2828):

```ml
| MultipleFlowAttributes ->
  [
    text "Unexpected ";
    code "@flow";
    text " declaration. Only one per ";
    text "file is allowed.";
  ]
```

**Comment by sam-rad** [comment]
What if we remove `// @flow` prepend from [flow operations](https://github.com/dotansimha/graphql-code-generator/blob/master/packages/plugins/flow/operations/src/index.ts#L38) since [flow plugin](https://github.com/dotansimha/graphql-code-generator/blob/c0f7e16f2e9ddaab61421643da7e4851cadccf97/packages/plugins/flow/flow/src/index.ts#L10) already has it?

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/flow-operations@1.18.2`.
